NJW/DAY14/오픈채팅방.js

```diff
@@ -0,0 +1,30 @@
+function solution(record) {
+    let answer = [];
+    let user_db = {};
+
+    for (const r of record) {
+        const word = r.split(" ");
+        const cmd = word[0];
+        const uid = word[1];
+        const nickname = word[2];
```

Comment:
```suggestion
        const [cmd, uid, nickname] = r.split(" ");
```
`구조 분해 할당`을 쓰면 중간 변수 없이 더 간결하게 작성할 수 있어요!